flexget/utils/imdb.py

```diff
@@ -239,10 +239,10 @@ def parse(self, imdb_id):
             return
 
         # Parse stuff from the title-overview section
-        name_elem = title_overview.find('h1', itemprop='name')
-        self.name = name_elem.find(text=True, recursive=False).strip()
+        name_elem = title_overview.find(itemprop='name')
+        self.name = name_elem.contents[0]
 
-        year = name_elem.find('a')
+        year = title_overview.find(attrs={'class': 'nobr'}).find('a', href=True)
```

Comment:
That is the span class that is wrapped around year. It was the only way I could make it work. I spent a fair amount of time on this since I don't use BS4. And it works on both style of pages the promotion pages and the standard pages. And even works on your no plot page. So If there's a better way I'd like to know, I couldn't figure one out.